pages/portfolio.html

```diff
@@ -1,12 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/grid.css">
+    <title>here is my profile</title>
 </head>
-<body>
-  
-</body>
+
+<div class="wrapper">
+    <div class="grid-container">
+        <header class="grid-header">
+        </header>
+
+        <nav class="grid-top-nav">
+            <ul>
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="about"><a href="about.html">About</a></li>
+                <li id="portfolio"><a href="portfolio.html">Portfolio</a></li>
+            </ul>
+        </nav>
+
+
+
+        <div class="grid-main">
+            <section class="grid-section">
+                    <img alt="Marjan's profile picture after being an excellent software engineer" src="images/myPic.jpg">
+            </section>
+            <section class="grid-section">
+                <h1 class="portfolio-h1">My Recent Projects:</h1>
+                <div class="portfolio-project-wrapper">
+                    <ol class="p-github-ul">
+                        <li class="p-github-li"><a class="portfolio-github-link" href="https://github.com/Marjan-abm/retro-video-store">Video Store</a></li>
+                        <ul class='p-desc-ul'>
```

Comment:
`<ul>` tags can not be nested inside of `<ol>` tags.  The MDN documentation has a "permitted content" section for each tag that identifies which tags can be nested inside of each tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol.